### PR TITLE
some fixes PySide6 6.1.0

### PR DESCRIPTION
**Pull request by pijyoi**
fix the easy parts of #1770 for PySide6 6.1.0.
i.e. RemoteGraphicsView.py still fails

### CODE REVIEW

File: pyqtgraph/dockarea/Dock.py [hunk 1]
```diff
@@ -198,7 +198,7 @@ def startDrag(self):
         self.drag.setMimeData(mime)
         self.widgetArea.setStyleSheet(self.dragStyle)
         self.update()
-        action = self.drag.exec_() if hasattr(self.drag, 'exec_') else self.drag.exec()
+        action = self.drag.exec() if hasattr(self.drag, 'exec') else self.drag.exec_()
```
Comment (ksunden) [inline pyqtgraph/dockarea/Dock.py:201]: this just simplifies to `self.drag.exec` now

Comment (pijyoi) [reply pyqtgraph/dockarea/Dock.py:201]: Qt.py provides Qt5 shim for QApplication.exec() but not for QDialog.exec()

Comment (ksunden) [reply pyqtgraph/dockarea/Dock.py:201]: I mean... the true and false cases are exactly the same in this diff

Comment (ksunden) [reply pyqtgraph/dockarea/Dock.py:201]: nevermind, I misread it

**Comment by j9ac9k** [comment]
thanks @pijyoi !  I'll try and take a closer look at the `RemoteProxy.py` error and see if I can get to the bottom of that one...

**Pull request merged**